<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostingWidget</class>
 <widget class="QWidget" name="PostingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="compressPathLayout_2">
       <item>
        <widget class="QLabel" name="compressPathLbl">
         <property name="text">
          <string>compress path: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="compressPathEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>temporary folder where the archives and par2 will be created (it will be cleaned once the post is done)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compressPathButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>select the temporary folder where the archives and par2 will be created (it will be cleaned once the post is done)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rarLayout">
       <item>
        <layout class="QHBoxLayout" name="rarPathLayout">
         <item>
          <widget class="QLabel" name="rarLbl">
           <property name="text">
            <string>rar path: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rarEdit">
           <property name="toolTip">
            <string>full path of the rar executable</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rarPathButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>select rar executable</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rarSizeLayout">
         <item>
          <widget class="QLabel" name="rarSizeLbl">
           <property name="text">
            <string>vol size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rarSizeEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>to split the rar archive in several volumes (0 to don't split)</string>
           </property>
           <property name="text">
            <string>42</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rarMaxCB">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>limit rar number</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="redundancyLayout">
       <item>
        <widget class="QLabel" name="redundancyLbl">
         <property name="text">
          <string>par2 redundancy (%): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="redundancySB"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SignedListWidget" name="filesList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selectFilesButton">
       <property name="toolTip">
        <string>Select the files to post (they may be compressed if needed) you can also right click on the files area just above</string>
       </property>
       <property name="text">
        <string>Select Files</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearFilesButton">
       <property name="toolTip">
        <string>remove all files</string>
       </property>
       <property name="text">
        <string>Remove All</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFolderButton">
       <property name="toolTip">
        <string>select a folder (only if you use compression)</string>
       </property>
       <property name="text">
        <string>Select Folder</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="nzbFileLayout">
       <item>
        <widget class="QLabel" name="nzbFileLbl">
         <property name="text">
          <string>nzb file: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nzbFileEdit">
         <property name="toolTip">
          <string>full path of the nzb file that would be created (check the nzbPath keyword in the config file to set the default directory)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nzbFileButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>select the nzb file absolute file name</string>
         </property>
         <property name="text">
          <string> ... </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="nzbPassLayout">
       <item>
        <widget class="QCheckBox" name="nzbPassCB">
         <property name="toolTip">
          <string>This should be the password of the archive you're posting</string>
         </property>
         <property name="text">
          <string>nzb password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nzbPassEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>password used in your archive that would also be added in the header of the nzb file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="passLengthSB">
         <property name="toolTip">
          <string>length of the password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="genPass">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>generate random password</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/genKey.png</normaloff>:/icons/genKey.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="compressOptionsLayout">
       <item>
        <widget class="QCheckBox" name="compressCB">
         <property name="toolTip">
          <string>compress the selected files using rar before posting</string>
         </property>
         <property name="text">
          <string>compress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="compressNameEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>archive name (file name obfuscation)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nameLengthSB">
         <property name="toolTip">
          <string>length of the archive name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="genCompressName">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>generate random archive name</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icons/genKey.png</normaloff>:/icons/genKey.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="par2CB">
         <property name="toolTip">
          <string>generate the par2 (the compress option must be selected)</string>
         </property>
         <property name="text">
          <string>gen par2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keepRarCB">
         <property name="toolTip">
          <string>by default archives and par2 files are deleted uppon post success but you can choose to keep them</string>
         </property>
         <property name="text">
          <string>keep archives</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="postLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="toolTip">
        <string>About ngPost</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="postButton">
       <property name="toolTip">
        <string>Let's Post!</string>
       </property>
       <property name="text">
        <string>Post Files</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/ngPost.png</normaloff>:/icons/ngPost.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="donateButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Donate</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/donate.png</normaloff>:/icons/donate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignedListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">hmi/SignedListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
